Exiled.Events/Patches/Fixes/GrenadesFailToExplodeFix.cs

```diff
@@ -0,0 +1,41 @@
+// -----------------------------------------------------------------------
+// <copyright file="GrenadesFailToExplodeFix.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Fixes
+{
+    using System.Collections.Generic;
+    using System.Reflection;
+    using System.Reflection.Emit;
+
+    using Grenades;
+
+    using HarmonyLib;
+
+    using UnityEngine;
+
+    /// <summary>
+    /// Fixes a Null Reference Exception being thrown caused by calling <see cref="Grenade.ServersideExplosion"/> from the coroutine <see cref="Grenade._Fuse(float)"/>.
+    /// </summary>
+    [HarmonyPatch(typeof(Grenade), nameof(Grenade.SetFuseTime))]
+    internal static class GrenadesFailToExplodeFix
+    {
+        private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions)
+        {
+            foreach (var inst in instructions)
```

Comment:
You should add some comments to explain what it does to non-devs people, everything else is fine.

Reply:
Sure!